app/lib/address_formatter.rb

```diff
@@ -0,0 +1,66 @@
+class AddressFormatter
```

Comment:
The govspeak gem has a feature called [`HCardPresenter`](https://github.com/alphagov/govspeak/blob/3027b9b3063f5495891e16ca830b909bbea43ce5/lib/govspeak/presenters/h_card_presenter.rb#L16), which seems to do much the same.

I wonder if it would be possible to use that, instead of copying over the formatter from Whitehall?

Reply:
That looks like the same thing, I'll have a look now thanks! 👀

Reply:
Would you mind having a second look at this? It looks like that worked ☝️. Only difference is what you can see on the image of the Iceland office which is probably inconsequential?

Reply:
This all looks good to me and will make these consistent with other parts of GOV.UK that have used the govspeak gem.